tests/geometric-multiscale/AxialGeoMultiscaleUniform.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<precice-configuration experimental="on">
+  <data:vector name="VelocityLike" />
+  <data:scalar name="PressureLike" />
+
+  <mesh name="Mesh1D" dimensions="3">
+    <use-data name="VelocityLike" />
+    <use-data name="PressureLike" />
+  </mesh>
+
+  <mesh name="Mesh3D" dimensions="3">
+    <use-data name="VelocityLike" />
+    <use-data name="PressureLike" />
+  </mesh>
+
+  <participant name="Fluid1D">
+    <provide-mesh name="Mesh1D" />
+    <receive-mesh name="Mesh3D" from="Fluid3D" />
+    <mapping:axial-geometric-multiscale
+      direction="read"
+      multiscale-type="collect"
+      multiscale-radius="1.0"
+      multiscale-axis="z"
+      from="Mesh3D"
+      to="Mesh1D"
+      constraint="consistent" />
+    <write-data name="VelocityLike" mesh="Mesh1D" />
+    <read-data name="PressureLike" mesh="Mesh1D" />
+  </participant>
+
+  <participant name="Fluid3D">
+    <receive-mesh name="Mesh1D" from="Fluid1D" />
+    <provide-mesh name="Mesh3D" />
+    <mapping:axial-geometric-multiscale
+      direction="read"
+      multiscale-type="spread"
+      multiscale-radius="1.0"
+      multiscale-axis="z"
+      spread-profile="uniform"
```

Comment:
Let's better keep the convention here and name it `multiscale-spread-profile`. We can later improve the config, but for now, this is the (suboptimal, yes) convention.